kong/plugins/response-ratelimiting/access.lua

```diff
@@ -36,21 +39,23 @@ local function get_usage(conf, identifier, limits, current_timestamp)
 
   for k, v in pairs(limits) do -- Iterate over limit names
     for lk, lv in pairs(v) do -- Iterare over periods
-      local current_usage, err = policies[conf.policy].usage(conf, identifier, k, lk, current_timestamp)
-      if err then
-        return nil, err
-      end
-
-      if not usage[k] then
-        usage[k] = {}
+      if lv ~= null then
```

Comment:
I don't fully understand why the `lv` can be `null`, but can it be `nil` too, thus?

```suggestion
      if lv ~= nil and lv ~= null then
```

Reply:
See https://github.com/Kong/kong/pull/8483#issuecomment-1057669512 .